### ISSUE TITLE
App_Plugin property editor using JSON valueType now wraps text field values in quotes

### ISSUE THREAD

**Issue by argentini** [umbraco/Umbraco-CMS#11989]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

9.3.0

### Bug summary

Since updating to 9.3.0, my App_Plugin property editor input field value is wrapping the value in quotation marks within the input tag. My App Plugin uses JSON as the editor value type. In my example, I have a YouTube editor that asks for the URL to the video. Since the upgrade my URL values now show as wrapped in quotation marks, and when I remove them and save, they reappear in the field even on page reload.

### Steps to reproduce

Create or use an App Plugin for a property editor that has a text input field where the editor is configured as valueType JSON.

---
_This item has been added to our backlog [AB#17434](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/17434)_

**Comment by markadrake** [comment]
Just for reference:

A number of changes happened with how JSON values are stored in the db.

PRs #11806 & #11904

**Comment by argentini** [comment]
@markadrake thanks for the quick response. I'm not sure how that applies to my issue. Setting a text value for an input field binding should not add quotation marks around the value. In the JS code I strip them and they still get saved. Is that by design?

**Comment by nul800sebastiaan** [comment]
I can reproduce going from v9.2.0 to v9.3.0 - I have an `App_Plugins/Issue11989` directory with the following files in it:

package.manifest
```json
{
    "propertyEditors": [
        {
            "alias": "TestEditor",
            "name": "Test",
            "icon": "icon-list",
            "group": "Common",
            "editor": {
                "view": "~/App_Plugins/Issue11989/test.html",
                "valueType": "JSON"
            }
        }
    ],
    "javascript": [
        "~/App_Plugins/Issue11989/test.controller.js"
    ]
}
```

test.controller.js
```javascript
angular.module('umbraco').controller('TestPluginController', function () {
});
```

test.html
```html
<div class="suggestion" ng-controller="TestPluginController">
    <p>{{model.value}}</p>
    <input type="text" ng-model="model.value" />
</div>
```

When I add this property to a document type in v9.2.0 it indeed returns the string `test`  without quotes around it but when I update that property after upgrading to v9.3.0 then we get quotes around the string:

https://user-images.githubusercontent.com/304656/153848753-ae312b51-e928-4a3d-a023-2505c526bf3c.mp4

**Comment by argentini** [comment]
@nul800sebastiaan I also noticed that when I navigate to a page and go to another tab, then save and publish, it wraps my input in backslashes. Yet, if I navigate to the same page and save from the tab that has the plugin input field, it saves the field value properly. It's like the act of navigating to another tab changes how the input field is saved.

**Comment by nikolajlauridsen** [comment]
Fixed in #12015
